safe/impact_functions/earthquake/pager_earthquake_fatality_model/metadata_definitions.py

```diff
@@ -72,6 +72,11 @@ def as_dict():
             'output': '',
             'actions': '',
             'limitations': [],
+            'map_title': '',
```

Comment:
Perhaps we can take the value from the ITB Earthquake Fatality IF. Same as previous comment